stl/inc/ranges

```diff
@@ -1140,16 +1140,18 @@ namespace ranges {
         // clang-format on
 
         _NODISCARD friend constexpr _Ioterator operator+(_Ioterator _It, const difference_type _Off) noexcept(
-            noexcept(static_cast<_Wi>(_It._Current + _Off))) /* strengthened */ requires _Advanceable<_Wi> {
-            return _Ioterator{static_cast<_Wi>(_It._Current + _Off)};
+            noexcept(_It += _Off)) /* strengthened */ requires _Advanceable<_Wi> {

```

Comment:
Pre-existing: This conditional `noexcept` is missing `is_nothrow_move_constructible_v<_Wi>`, although now I think it would be more clearly spelled `is_nothrow_move_constructible_v<_Ioterator>` (they are equivalent).

Mea culpa, I'll fix this.